org.eclipse.languageserver/src/org/eclipse/languageserver/LanguageServiceAccessor.java

```diff
@@ -111,14 +114,34 @@ public boolean equals(Object obj) {
 		public final URI fileUri;
 		public final IFile file;
 		public final IDocument document;
-		public final LanguageClientEndpoint languageClient;
+		private final LanguageClientEndpoint languageClient;
```

Comment:
mixing public field and getters/setters is not recommended if we want the API to be uniform. Either keep it public and remove getters/setters or wrap them all in getters/setters.